src/themes/oneui/placeholders/_input.scss

```diff
@@ -23,11 +23,11 @@
 
     &:focus,
     &:hover {
-        border-color: var(--color-brand);
+        border-color: var(--color-primary);
     }
 
     &::placeholder {
-        color: var(--color-muted);
+        color: var(--color-light);
```

Comment:
same as above (`neutral` vs `light`)

Reply:
we checked all changes about colors with Lidiia last week, so, it should be correct :)